fixtures/mock-editor.js

```diff
@@ -0,0 +1,38 @@
+import assert from 'assert';
```

Comment:
Why mock the editor instead of just using it?

See: https://github.com/sindresorhus/atom-editorconfig/blob/6dd0a62d4d8bac118414add15409fd2093637761/spec/iss108-spec.js#L24-L25